tests/unit/ssg-module/test_checks.py

```diff
@@ -32,14 +32,23 @@ def test_get_oval_path():
                                  "missing_ovals")
 
 
-def test_is_cce_valid():
-    icv = ssg.checks.is_cce_valid
+def test_is_cce_format_valid():
+    icv = ssg.checks.is_cce_format_valid
     assert icv("CCE-27191-6")
     assert icv("CCE-7223-7")
 
     assert not icv("not-valid")
     assert not icv("1234-5")
-    assert not icv("12345-6")
     assert not icv("TBD")
     assert not icv("CCE-TBD")
     assert not icv("CCE-abcde-f")
+
+
+def test_is_cce_value_valid():
+    icv = ssg.checks.is_cce_value_valid
+    assert icv("CCE-27191-6")
+    assert icv("CCE-7223-7")
```

Comment:
This needed to be `CCE-27223-7` like in #3820, so I pushed an update.

Reply:
Thanks @cipherboy